### DISCUSSION TITLE
Add Node container status output to Grid page

### DISCUSSION THREAD

**Discussion by innovate-support** [Security-Onion-Solutions/securityonion#2999] [Ideas]
It would be most helpful if the Grid webpage also showed the 'Status' results of each node's containers instead of just online/offline.  Although online/offline is good to know, it doesn't tell you anything about the actual functionality of the node.

**Comment by jertel** [comment]
This is planned for a future release, and is why the current status is showing 'Online' in blue. When we implement the next phase to this it will show a green 'OK' status, representing the node being in a healthy state.
